internal/buildscripts/packaging/msi/splunk-otel-collector.wxs

```diff
@@ -39,8 +39,10 @@
                      <RegistryKey Root="HKLM" Key="SYSTEM\CurrentControlSet\Services\EventLog\Application\splunk-otel-collector">
                         <RegistryValue Type="expandable" Name="EventMessageFile" Value="%SystemRoot%\System32\EventCreate.exe"/>
                      </RegistryKey>
-                     <RegistryKey Root="HKLM" Key="SYSTEM\CurrentControlSet\Control\Session Manager\Environment">
-                        <RegistryValue Type="string" Name="SPLUNK_CONFIG" Value="[CommonAppDataFolder]Splunk\OpenTelemetry Collector\agent_config.yaml"/>
+                     <RegistryKey Root="HKLM" Key="SYSTEM\CurrentControlSet\Services\splunk-otel-collector">
```

Comment:
Would adopting this path [here](https://github.com/signalfx/splunk-otel-collector/blob/52ec41e2119ea6db9478bc645ac4a38d9c628359/internal/buildscripts/packaging/installer/install.ps1#L169) be necessary for these changes? not sure if you've already done this at some point in this branch.

Reply:
Yes, this those paths will need to be update, however, notice that the env vars for the instrumentation itself will stay on that path. In this PR it is done here https://github.com/signalfx/splunk-otel-collector/pull/3930/files#diff-a0762d7dac27b79c910fcb6bf782cadc7428dd4574d065afeabc5ea90eb9538dL583